bob/main.py

```diff
@@ -256,6 +260,19 @@ def replace_weather_description_with_emojis(description):
         description = description.replace(i,j) 
     return description
 
+
+def low_probability_reply(update, context, int): # added int argument for unit testing
+    if int is None:
+        random_int = secrets.randbelow(10000) # 0,01% probability
```

Comment:
Käytä sitä randomia. Se sonar juttu on false positive ja siihen pystyy klikkaamaan vaan "not an issue".

Reply:
Selvä! Halusin vaan olla ärsyttävän pedantti lol